src/ai/backend/manager/registry.py

```diff
@@ -777,6 +777,10 @@ async def enqueue_session(
 
         mounts = kernel_enqueue_configs[0]['creation_config'].get('mounts') or []
         mount_map = kernel_enqueue_configs[0]['creation_config'].get('mount_map') or {}
+        root_mounts = list(set(map(lambda x: x.split("/")[0], mounts)))
+        root_mount_map = dict()
+        for item in root_mounts:
+            root_mount_map[item] = list(filter(lambda x: x.find(item) >= 0, mounts))
```

Comment:
How about rewriting this using the list comprehension syntax?

Reply:
Could you explain what is the purpose of `root_mount_map` variable?

Reply:
`root_mount_map` is not necessary, so it is deleted. Now `mount` play this role.